contrib/cirrus/postbuild.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -eo pipefail
+
+# This script attempts to confirm all included go modules from
+# other sources match what is expected in `vendor/modules.txt`
+# vs `go.mod`.  Also make sure that the generated bindings in
+# `pkg/bindings/...` are in sync with the code.  It's intended
+# for use after successfully building podman, to prevent wasting
+# time on tests that might otherwise succeed with bad/ugly/invalid
+# code.
+
+source /etc/automation_environment
+source $AUTOMATION_LIB_PATH/common_lib.sh
+
+# There's no need to perform further checks on more than one
+# CI platform.  These variables are defined in .cirrus.yml
+# shellcheck disable=SC2154
+if [[ ! "${DISTRO_NV}" =~ ${FEDORA_NAME} ]]; then
+    echo "Skipping additional checks on $DISTRO_NV"
+    exit 0
+fi
+
+make install.tools
+make vendor
```

Comment:
Another place where `echo running make vendor / generate-bindings / completions` progress messages would be appreciated by a future hapless investigator.

Reply:
Yep, I've got a library function `showrun()` that behaves like single-line `set -x`.  That should do the trick.